doc/specs/stdlib_sorting.md

```diff
@@ -196,6 +198,18 @@ magnitude slower than `ORD_SORT`. Its memory requirements are also
 low, being of order O(Ln(N)), while the memory requirements of
 `ORD_SORT` and `SORT_INDEX` are of order O(N).
 
+#### The `RADIX_SORT` subroutine
+
+`RADIX_SORT` is a implementation of LSD [radix sort](https://en.wikipedia.org/wiki/Radix_sort),
+using `256` (one byte) as the radix. It only works for fixed width data,
```

Comment:
One byte is 8 bits, right? Why is "one byte" mentioned here?

Reply:
> One byte is 8 bits, right? Why is "one byte" mentioned here?

Yes, one byte is not 8 bits in some machine, I will remove it.
```suggestion
using `256` as the radix. It only works for fixed width data,
```